.github/workflows/build_and_test.yml

```diff
@@ -871,6 +871,9 @@ jobs:
         python3.9 -m pip install sphinx_plotly_directive 'numpy>=1.20.0' pyarrow pandas 'plotly>=4.8'
         python3.9 -m pip install 'docutils<0.18.0' # See SPARK-39421
     - name: Install dependencies for documentation generation
+      # Keep the version of Bundler here in sync with the following locations:
+      #   - dev/create-release/spark-rm/Dockerfile
+      #   - docs/README.md
```

Comment:
nit. Shall we move these comment before the line, `gem install bundler -v 2.4.22`, because that is more closer and clear.

Reply:
To be clear, do you mean I should move this block of comments one line down, just below the `run: |`?

Reply:
Yes~